tao/tao_bridge/common.cc

```diff
@@ -45,8 +45,8 @@ static void AllocateTaoBridgeFlags() {
                               &opts->verbose_compilation_err_log));
   CHECK_OK(ReadBoolFromEnvVar("TAO_ENFORCE_VERBOSE_COMPILATION_LOG", false,
                               &opts->verbose_compilation_log));
-  CHECK_OK(ReadStringFromEnvVar("TAO_CACHE_DUMP_PATH", "",
-                                &opts->tao_cache_dump_path));
+  CHECK_OK(ReadStringFromEnvVar("DISC_COMPILATION_CACHE_DUMP_PATH", "",
+                                &opts->disc_cache_dump_path));
```

Comment:
why some remains TAO_* and some becomes DISC_*, what is the standard here?

Reply:
For the new envs, better to use DISC_*.  For the old envs, we gradually rename it to DISC_* on demand？